make/tools.mk

```diff
@@ -1,29 +1,17 @@
-CTR=docker
-
-# We don't need to re-download the whole Go distribution, but we still want
-# to make sure that we don't clutter the PATH by using a symlink. So we use
-# a symlink of the system Go in bin/tools.
-HELM=$(PWD)/bin/tools/helm
-COSIGN=$(PWD)/bin/tools/cosign
-CMREL=$(PWD)/bin/tools/cmrel
-YQ=$(PWD)/bin/tools/yq
-KUBECTL=$(PWD)/bin/tools/kubectl
-KIND=$(PWD)/bin/tools/kind
-CRANE=$(PWD)/bin/tools/crane
```

Comment:
The comment was outdated. These env vars were never used anywhere.

Reply:
Some of these were used, and their uses were deleted in this PR, e.g. for `yq`: https://github.com/cert-manager/cert-manager/pull/4988/files#diff-7fad63ce343e43adef92cb03937db29d2f40b2c1d8c0b0de84735176ec697bf2L88-R88

The intent was that these env vars would allow for a user to sub in a different version of the binary by choice